src/index.html

```diff
@@ -14,13 +14,13 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-<form action="" class="searchbar-1" data-qa="big">
-  <label for="searchbar-1__input" class="search-label-1"></label>
-  <input type="text" class="searchbar-1-input input" id="searchbar-1__input" placeholder="Try “Los Angeles”" data-qa="keypress" />
+<form action="#" class="search-input-form" data-qa="big">
+  <label for="searchbar-1__input" class="search-label-big"></label>
```

Comment:
If you try to write class by BEM, look to the documentation - https://ru.bem.info/methodology/quick-start/
block - `block or bl-ok`
element - `block__element`
modification - `block__element_modification`